### PR TITLE
changed to allow for sending to just organizer

### PR DESCRIPTION
**Pull request by WhoShotCupidd**
pretty quick and easy way to send to just the organizer or the whole event

### CODE REVIEW

File: code/noRAM/app/src/main/java/com/example/noram/model/PushNotificationService.java [hunk 1]
```diff
@@ -77,14 +79,16 @@ public void onMessageReceived(RemoteMessage remoteMessage) {
      * @param title the title of the notification
      * @param data the data of the notification
      * @param event the event to send the notification to
+     * @param isOrganizer a boolean to check if we wish to only send the notification to the organizer
      */
-    public void sendNotification(String title, String data, Event event) {
+    public void sendNotification(String title, String data, Event event, Boolean isOrganizer) {
```
Comment (ColeDewis) [inline code/noRAM/app/src/main/java/com/example/noram/model/PushNotificationService.java]: "isOrganizer" is a bit of a confusing variable name here imo, but its not a _huge_ deal

**Review by ColeDewis — APPROVED** [review]
see comment but either way looks fine

File: code/noRAM/app/src/main/java/com/example/noram/model/PushNotificationService.java (hunk 1 above)
Comment (WhoShotCupidd) [reply code/noRAM/app/src/main/java/com/example/noram/model/PushNotificationService.java]: anything u think is better?

Comment (WhoShotCupidd) [reply code/noRAM/app/src/main/java/com/example/noram/model/PushNotificationService.java]: I changed it

**Pull request merged**